api/validators/FeedbackControllerRequests.ts

```diff
@@ -1,27 +1,44 @@
 import { Type } from 'class-transformer';
-import { IsDefined, IsNotEmpty, MinLength, ValidateNested } from 'class-validator';
+import { Allow, IsDefined, IsNotEmpty, MinLength, ValidateNested } from 'class-validator';
 import { IsValidFeedbackType, IsValidFeedbackStatus } from '../decorators/Validators';
 import {
   SubmitEventFeedbackRequest as ISubmitEventFeedbackRequest,
   SubmitFeedbackRequest as ISubmitFeedbackRequest,
   UpdateFeedbackStatusRequest as IUpdateFeedbackStatusRequest,
+  FeedbackSearchOptions as IFeedbackSearchOptions,
   Feedback as IFeedback,
   FeedbackType,
   FeedbackStatus,
 } from '../../types';
+import { UserModel } from '../../models/UserModel';
+import { EventModel } from '../../models/EventModel';
 
 export class Feedback implements IFeedback {
   @IsDefined()
   @IsNotEmpty()
-  title: string;
+  user: UserModel;
 
   @IsDefined()
-  @MinLength(100)
+  @IsNotEmpty()
+  event: EventModel;
```

Comment:
Same concept here as the Users one above, specifically about passing in the entire public Event object. This makes API requests bloated to read and sends over far more data than we need. This should just be the Event UUID, and we'd need to query for the event in the service layer